coins/monero/src/wallet/seed/mod.rs

```diff
@@ -67,26 +67,52 @@ impl Seed {
       CLASSIC_SEED_LENGTH | CLASSIC_SEED_LENGTH_WITH_CHECKSUM => {
         ClassicSeed::from_string(words).map(Seed::Classic)
       }
+      POLYSEED_LENGTH => PolySeed::from_string(words).map(Seed::PolySeed),
       _ => Err(SeedError::InvalidSeedLength)?,
     }
   }
 
   /// Create a Seed from entropy.
-  pub fn from_entropy(lang: Language, entropy: Zeroizing<[u8; 32]>) -> Option<Seed> {
-    ClassicSeed::from_entropy(lang, entropy).map(Seed::Classic)
+  pub fn from_entropy(seed_type: SeedType, entropy: Zeroizing<[u8; 32]>) -> Option<Seed> {
```

Comment:
My intent for the entropy function was for it to return the bytes usable as a key (it being a rename of spend_key which is somewhat inaccurate under Polyseed). I now realize I've messed up in that the entropy *is* defined as the original 19-bytes?

We should add a key function to return the generated *key*. This would be the entropy for classic seeds yet the PBKDF output for Polyseed.